src/log/zipkin.cpp

```diff
@@ -187,6 +212,8 @@ void zipkin::impl::log( zipkin_span::span_data&& span ) {
       wlog( "unable to connect to zipkin: ${u}, error: unknown", ("u", zipkin_url) );
    }
    ++consecutive_errors;
+   connected = false;
+   sleep(30);
```

Comment:
Threads continue to post to the boost asio queue while this sleeps which could cause memory issues.

Reply:
Moved sleep(30) into the calling thread of post.